packages/flutter_blue_plus_linux/lib/flutter_blue_plus_linux.dart

```diff
@@ -946,6 +942,7 @@ final class FlutterBluePlusLinux extends FlutterBluePlusPlatform {
           characteristicUuid: Guid.fromBytes(
             characteristic.uuid.value,
           ),
+          characteristicIndex: 0,
```

Comment:
Should this be the `characteristicIndex` from the `request` instead of `0`?

```suggestion
          characteristicIndex: request.characteristicIndex,
```

Reply:
we're not trying to pass it back.